### PR TITLE
create empty avatar if no text are provided

### PR DESCRIPTION
**Pull request by MariuszHenn**

### CODE REVIEW

**Comment by ArturT** [comment]
Could you fix tests on CI?

**Comment by MariuszHenn** [comment]
Can You provide me imagemagic library version
Available on CI? Rspec on my configurstion passes without any error.

Could you fix tests on CI?

—
Reply to this email directly or view it on GitHub
<https://github.com/lucek/avatarly/pull/12#issuecomment-159202557>.

File: spec/avatarly_spec.rb [hunk 1]
```diff
@@ -61,6 +61,12 @@
         assert_image_equality(result, :H_black_white_32)
       end
 
+      it 'does not break if no text found' do
+        result = described_class.generate_avatar(nil,
+                                                 background_color: "#000000")
+        assert_image_equality(result, :black_empty, 34)
```
Comment (ArturT) [inline spec/avatarly_spec.rb]: you can try with higher threshold than 34 then tests should pass

**Comment by MariuszHenn** [comment]
everything is ready to go

**Comment by ArturT** [comment]
Thanks for PR!

**Pull request merged**